packages/components/src/components/code-snippet/_code-snippet.scss

```diff
@@ -309,6 +310,7 @@
   }
 
   .#{$prefix}--snippet--inline .#{$prefix}--btn--copy__feedback {
+    top: calc(100% - #{rem(4px)}); // offset
```

Comment:
What does the 4px value relate to?

Reply:
it's the default offset of the tooltip caret for the inline code snippet. all of the positioning values for the copy button tooltip are arbitrary hard coded rem values so unless we are ok with refactoring the current styles for the copy button base or just code snippet copy button, this seemed like the stop gap solution until we can use our updated tooltips under the hood. the alternative I went with initially was a percent based offset (`top: 80%`) that made the gap 4px at the default size ref https://github.com/carbon-design-system/carbon/pull/4502#discussion_r340873899

Reply:
Would you want to leave a quick note for where it comes from or why it's used? Just to help with debugging in the future